src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -674,14 +680,21 @@ private void Cancel(bool rollbackTemporaryEdits)
         }
 
         public void Commit(bool previewChanges = false)
+            => CommitWorker(previewChanges);
```

Comment:
What's with the worker method here?

Reply:
i would like to validate from tests taht we actually canceled.